### ISSUE TITLE
GPL-623 Automatically update locations of sorted picks

### ISSUE THREAD

**Issue by JamesGlover** [sanger/sequencescape#2888]
**User story**
As a user I would like to be able to automatically scan in all plates without picks from a plate_pick session (GPL588 #2860) into a defined labwhere location.

**Who are the primary contacts for this story**
James G
Eloise W

**Acceptance criteria**
To be considered successful the solution must allow:
- [ ] Provide a field for scanning a labwhere location
- [ ] Provide an option for transferring all scanned plates without registered picks to this location
- [ ] Should probably provide a means to distinguish between 'Not found' and 'No pick' plates, as the former probably have no positives, whereas the latter just haven't had picks created yet.

Suggested nice to haves:
- [ ] Provide means of unticking plates to specifically exclude them
- [ ] Provide dynamic feedback of update progress (Unless labwhere has bulk updates?)

**Dependencies**
This story is blocked by the following dependencies:
- #<issue_no.>
- sanger/<repo>#<issue_no.>

**Additional context**

**Comment by JamesGlover** [comment]
This came from UAT feedback from GPL-588 and provides tracking of the scanned plates. The separate labwhere scan was causing mis-scans and they wanted an extra layer of tracking.

**Comment by rl15** [comment]
Rich L wrote (Tuesday, 15 December 2020 at 14:47)
Eloise, Q> Does this backlog item still have high value to do?
